src/phpDocumentor/Descriptor/ConstantDescriptor.php

```diff
@@ -52,13 +52,13 @@ public function setParent($parent)
             ? $parent->getFullyQualifiedStructuralElementName() . '::' . $this->getName()
             : $this->getName();
 
-        $this->setFullyQualifiedStructuralElementName($fqsen);
+        $this->setFullyQualifiedStructuralElementName(new Fqsen($fqsen));
```

Comment:
All `*etFullyQualifiedStructuralElementName()` methods now yield and accept Fqsen objects; apparently this was already partly the case in many locations but with the added type-hinting it became painfully obvious. This went well in the past because without declare strict, the Fqsen was converted into a string